scripts/v.import/v.import.py

```diff
@@ -121,6 +123,48 @@ def cleanup():
         grass.try_remove(SRCGISRC)
 
 
+def checkGDALversion(smallerversion="2.4.1"):
+    """Checks GDAL version if it is smaller than given smaller version
+
+    :param smallerversion: version number to check if the actual GDAL version
+                           is smaller
+    :type smallerversion: str
+    """
```

Comment:
Please use underscores in names. I don't think the default argument is appropriate in this case; move the value to the actual call. I think "smaller" is not the right wording (smaller refers to size) , use "lower".

Reply:
You could also revise the naming (check what about the GDAL version?) and, e.g., split that into `gdal_version()` and `version_less_than()`.